.gitignore

```diff
@@ -3,3 +3,7 @@
 /build/
 /dist/
 /git-multimail/*.egg-info
+
+# The Vi/Vim temporary files
+*.swp
```

Comment:
Thanks. I didn't know Git could have global .gitignore files. Set it up into my portable git configuration immediately.
Pushed again by force.
Bad is the day when one doesn't learn new things about Git...